components/paging/pager-load-more.js

```diff
@@ -102,11 +103,11 @@ class LoadMore extends FocusMixin(FocusVisiblePolyfillMixin(LocalizeCoreElement(
 				<span class="d2l-offscreen" role="alert">${this.localize('components.pager-load-more.status-loading')}</span>
 				<d2l-loading-spinner size="24"></d2l-loading-spinner>
 			` : html`
-				<span class="action">${this.localize('components.pager-load-more.action', { count: this.pageSize })}</span>
+				<span class="action">${this.localize('components.pager-load-more.action', { count: formatNumber(this.pageSize) })}</span>
```

Comment:
While this produces the result that I want because I am only ever passing integer values greater than zero, it looks like passing formatted numbers into the localization lib can potentially be problematic. Given a lang-term that is using the plural type syntax:
```
{totalCount, plural, one {{showingCount} of {totalCount} item} other {{showingCount} of {totalCount} items}}
```

In Danish:
* `this.localize('components.pager-load-more.action', { count: 0.1 }` yields the singular format (correct)
* `this.localize('components.pager-load-more.action', { count: formatNumber(0.1) })` yields the other format (incorrect)

This is because `formatNumber` for Danish yields "0,1", which does not match the singular values:

![image](https://user-images.githubusercontent.com/9042472/181576866-0a4c94af-82c3-42ff-9505-3ef875874d36.png)

As mentioned, this isn't problematic for the pager per se, but passing formatted values seems incompatible with plural syntax types in ICU messages @dlockhart , @bearfriend , do you have any insight into how such a scenario should be handled?

Reply:
Yikes yeah, that's not great. I'm guessing with larger numbers that might result in group separators (e.g. `1,000,000`) we could be in trouble too.

I wonder if the `pluralize` function itself does any number formatting on its own? We don't use [the number type](https://formatjs.io/docs/core-concepts/icu-syntax#number-type) because obviously we have our own number formatting rules, but `pluraize` might use it under the cover.

Reply:
I think passing in formatted numbers to the subject of the `plural` is probably a bad idea. Let it do its logic on `totalCount` but use a formatted variable for rendering:
```
{totalCount, plural, one {{showingCount} of {totalCountFormatted} item} other {{showingCount} of {totalCountFormatted} items}}
```
Then:
```
this.localize('components.pager-load-more.info', { showingCount: ..., totalCount: 0.1, totalCountFormatted: formatNumber(0.1) })
```

Reply:
That's an excellent suggestion!

Reply:
It's really just the same as this, but up one layer since we have to use our own formatting:
```
{totalCount, plural, one {{showingCount} of {totalCount, number} item} other {{showingCount} of {totalCount, number} items}}
```

We probably could make formatters from our custom stuff to use this way.

Reply:
Yep, that's excellent.  Much thanks! 

Adapting our custom formatting to something it can use might be an interesting thing to look into in the future.